protoc-gen-twirp_php/templates/service/Client.php

```diff
@@ -70,6 +75,22 @@ public function __construct(
         $this->requestFactory = $requestFactory;
         $this->streamFactory = $streamFactory;
     }
+
+    /**
+     * Returns a new client configured to do json request instead of doing protobuf request which is default and recommended.
```

Comment:
```suggestion
     * Returns a new client configured to use JSON serialization instead of Protobuf.
     *
     * Twirp servers understand both Protobuf and JSON, but for production systems Protobuf is recommended.
     * JSON is available for development purposes.
     *
     * @see https://twitchtv.github.io/twirp/docs/proto_and_json.html
```